pkg/tlshelpers/client_lister.go

```diff
@@ -0,0 +1,77 @@
+package tlshelpers
+
+import (
+	"context"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	v1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+)
+
+// Below lister implementations are required to make the library-go certificate logic more robust.
+// Especially in render, we were relying on a race condition between the informer cache and the client
+// to create the bundles correctly. This lister always queries the API (or its mock) to directly return the
```

Comment:
Is this something that we need to fix back in library-go? Or is it only us that's impacted with our use of the render cmd?

Reply:
I think this is a side-effect of the fake libs in upstream k8s